specs/ExtendedAcceleratorKeyPressed.md

```diff
@@ -0,0 +1,182 @@
+    // All browser accelerator keys are disabled with this setting. The default value for the 
+    // `AllowBrowserAcceleratorHandling`property is set to the current value of this setting. 
+    // Changing this setting to `false` will change the default setting for 
+    // AllowBrowserAcceleratorHandling to `false`
+    _webViewSettings.AreBrowserAcceleratorKeysEnabled = false;
+
+    webView.CoreWebView2.AcceleratorKeyPressed += WebView2Controller_AcceleratorKeyPressed;
+
+    void WebView2Controller_AcceleratorKeyPressed(object sender, CoreWebView2AcceleratorKeyPressedEventArgs e)
+    {
+        switch (e.KeyEventKind)
+        {
+            case CoreWebView2KeyEventKind.KeyDown:
+            case CoreWebView2KeyEventKind.SystemKeyDown:
+                {
+                    // Allow the browser to process F7 key
+                    if (e.VirtualKey == Key.F7) 
+                    {
+                        e.AllowBrowserAcceleratorHandling = true;
+                    }
+                    break;
+                }
+            
+        }
+    }
+
+```
+
+# Remarks
+By default, `AreBrowserAcceleratorKeysEnabled` is `TRUE` and default value for `AllowBrowserAcceleratorHandling`
+is `TRUE`. When developers set `AreBrowserAcceleratorKeysEnabled` to `FALSE` this will also change the default 
+value for `AllowBrowserAcceleratorHandling` to `FALSE`. If developers want specific keys to be handled by the 
+browser after changing the `AreBrowserAcceleratorKeysEnabled` settings to `FALSE`, they need to enable them by 
+setting `event_args->put_ AllowBrowserAcceleratorHandling(TRUE)`. This option will give the event arg higher 
+priority over the settings when the browser decide whether to handle the keys.  
+
+# API Details
+## C++
+```
+/// This is a continuation of the ICoreWebView2AcceleratorKeyPressedEventArgs interface.
+[uuid(45238725-3774-4cfe-931f-7985a1b5866f), object, pointer_default(unique)]
+interface ICoreWebView2AcceleratorKeyPressedEventArgs2 : ICoreWebView2AcceleratorKeyPressedEventArgs {
+  /// Browser accelerator keys are the keys/key combinations that access features specific to 
+  /// a web browser, including but not limited to:
+  ///  - Ctrl-F and F3 for Find on Page
+  ///  - Ctrl-P for Print
+  ///  - Ctrl-R and F5 for Reload
+  ///  - Ctrl-Plus and Ctrl-Minus for zooming
+  ///  - Ctrl-Shift-C and F12 for DevTools
+  ///  - Special keys for browser functions, such as Back, Forward, and Search
+  ///
+  /// These APIs allow developers to enable/disable the browser from handling a specific browser 
+  /// accelerator key. These APIs do not disable accelerator keys related to movement and text 
+  /// editing, such as:
+  ///  - Home, End, Page Up, and Page Down
+  ///  - Ctrl-X, Ctrl-C, Ctrl-V
+  ///  - Ctrl-A for Select All
+  ///  - Ctrl-Z for Undo
+  ///
+  /// The exsiting `AreBrowserAcceleratorKeysEnabled` API is a convenience setting for developers 

```

Comment:
Throughout this ref doc comment please make sure you specify the class on which the property or event resides. Also for example applies to AcceleratorKeyPressed - should say CoreWebView2Controller.AcceleratorKeyPressed (is that the right class?)
 
```suggestion
  /// The exsiting `CoreWebView2Settings.AreBrowserAcceleratorKeysEnabled` API is a convenience setting for developers 

```